tgui/packages/tgui/interfaces/EFTPOS.js

```diff
@@ -21,94 +36,82 @@ export const EFTPOS = (props, context) => {
+      <Box mt={.5} fontSize="1.25rem" align="center" justify="center">
         {transaction_paid
           ? 'This transaction has been processed successfully '
           : 'Swipe your card to finish this transaction.'}
-      </NoticeBox>
-    </Section>
+        </Box>
+    </>
   );
 };
 
 const UnlockedView = (props, context) => {
   const { act, data } = useBackend(context);
-  const { transaction_purpose, transaction_amount, linked_account } = data;
+  const [searchText, setSearchText] = useLocalState(context, 'searchText', '');
+  const {
+    transaction_purpose,
+    transaction_amount,
+    linked_account,
+    available_accounts
+  } = data;
+
+  let accountMap = []
+  available_accounts.map(account => (
+    accountMap[account.name] = account.UID
+  ))
   return (
-    <Section title="Transation Settings" mt={1}>
-      <LabeledList>
-        <LabeledList.Item label="Transaction Purpose">
-          <Button
-            content={transaction_purpose}
-            icon="edit"
-            onClick={() => act('trans_purpose')}
-          />
-        </LabeledList.Item>
-        <LabeledList.Item label="Value">
-          <Button
-            // Ternary required otherwise the 0 is offset weirdly
-            content={transaction_amount ? transaction_amount : '0'}
-            icon="edit"
-            onClick={() => act('trans_value')}
-          />
-        </LabeledList.Item>
-        <LabeledList.Item label="Linked Account">
-          <Button
-            content={linked_account ? linked_account : 'None'}
-            icon="edit"
-            onClick={() => act('link_account')}
-          />
-        </LabeledList.Item>
-        <LabeledList.Item label="Actions">
-          <Button
-            content="Lock in new transaction"
-            icon="lock"
-            onClick={() => act('toggle_lock')}
-          />
-          <Button
-            content="Change access code"
-            icon="key"
-            onClick={() => act('change_code')}
-          />
-          <Button
-            content="Reset access code"
-            tooltip="Requires Captain, HoP or CC access"
-            icon="sync-alt"
-            onClick={() => act('reset')}
-          />
-        </LabeledList.Item>
-      </LabeledList>
-    </Section>
+    <LabeledList>
+      <LabeledList.Item label="Transaction Purpose">
+        <Button
+          content={transaction_purpose}
+          icon="edit"
+          onClick={() => act('trans_purpose')}
+        />
+      </LabeledList.Item>
+      <LabeledList.Item label="Value">
+        <Button
+          // Ternary required otherwise the 0 is offset weirdly
+          content={transaction_amount ? transaction_amount : '0'}
+          icon="edit"
+          onClick={() => act('trans_value')}
+        />
+      </LabeledList.Item>
+      <LabeledList.Item label="Linked Account">
+        <Box>{linked_account.name}</Box>
+        <Input
+          placeholder="Search by account name"
```

Comment:
This box text is cut off, as seen in your screenshot

Reply:
fixed